deps/zlib/zconf.h

```diff
@@ -17,7 +17,7 @@
 #if !defined(CHROMIUM_ZLIB_NO_CHROMECONF)
 /* This include does prefixing as below, but with an updated set of names.  Also
  * sets up export macros in component builds. */
-//#include "chromeconf.h"
+#include "chromeconf.h"
```

Comment:
This change still needs to be backed out IIRC. Either that, or you might try adding defining `CHROMIUM_ZLIB_NO_CHROMECONF` in the gyp file to see if that works. I commented this out in my original PR that added the Chromium zlib implementation because it was breaking zlib tests.